.github/workflows/ci.yml

```diff
@@ -60,7 +60,7 @@ jobs:
 
     strategy:
       matrix:
-        go: [1.19.x, 1.20.x]
+        go: [1.20.x]
```

Comment:
should we also check 1.21 and/or 1.22.4?  noting we moved to actions/setup-go@v5 in core..

Reply:
Okay, added another commit to bump setup-go to v5, too.

Reply:
Agreed, I think it makes sense to _either_ test min (1.20) and latest (1.22), or if CI is short to test all of 1.20, 1.21, 1.22.